deployer/auth.py

```diff
@@ -150,4 +179,7 @@ def get_client_creds(self, client, connection_name):
             'scope': ['openid', 'name', 'profile', 'email']
         }
 
+        if connection_name == 'password':
```

Comment:
Is this different than for other type of connections?

Reply:
If we want to logout everyone at the Auth0 session layer regardless of the connection type, then it shouldn't be different.
I just removed the condition. Ty! :D